SConstruct

```diff
@@ -75,10 +116,10 @@ env.Alias("external", external)
 # pipeline
 ingest = env.Alias("ingest", raws + visits)
 
-num_process = GetOption('num_jobs')
+num_jobs = GetOption('num_jobs')
```

Comment:
Why the name change?

Reply:
Oh, my bad, I wanted to put all parameter reading on top, but then when that didn't work out as neatly as I assumed it would, I reverted back and this is the side-effect.